src/DynamicMvvm.Abstractions/ViewModel/IViewModel.Extensions.Children.cs

```diff
@@ -32,13 +32,18 @@ public static partial class IViewModelExtensions
 		/// <param name="viewModel">The parent <see cref="IViewModel"/>.</param>
 		/// <param name="childViewModelProvider">The factory to the child <see cref="IViewModel"/>.</param>
 		/// <param name="name">The child ViewModel's name.</param>
-		/// <returns>The attached child <see cref="IViewModel"/>.</returns>
+		/// <returns>The attached child <see cref="IViewModel"/>. Default of <typeparamref name="TChildViewModel"/> is returned if the <see cref="IViewModel"/> is disposed.</returns>
 		public static TChildViewModel GetChild<TChildViewModel>(
 			this IViewModel viewModel,
 			Func<TChildViewModel> childViewModelProvider,
 			[CallerMemberName] string name = null
 		) where TChildViewModel : IViewModel
 		{
+			if (viewModel.IsDisposed)
```

Comment:
It's the type of reasoning that should be documented somewhere, maybe we should start a ArchitectureLogs.md and note the fact that we realised a simple "get" from the view could recreate disposables on disposed viewmodels.